src/extensions/sessionrecording.js

```diff
@@ -24,6 +25,19 @@ export class SessionRecording {
         }
     }
 
+    toggleSessionRecording(turnOn) {
```

Comment:
I think more explicit methods are better. E.g. `started()`, `start()` and `stop()`. As-is, it feels like this method cannot be reliably used to turn session recording off.